components/tabs/tab-internal.js

```diff
@@ -44,11 +44,7 @@ class Tab extends SkeletonMixin(RtlMixin(LitElement)) {
 				top: 0.15rem;
 			}
 			:host(:first-child) .d2l-tab-text {
-				margin-left: 0;
-			}
-			:host([dir="rtl"]:first-child) .d2l-tab-text {
-				margin-left: 0.6rem;
```

Comment:
`margin-right` for first child for LTR is actually `0.5rem` so this is inconsistent. Let me know if there was a reason why that I'm missing.

Reply:
Can we remove `RtlMixin` entirely from some of these are is the `dir` attribute still needed for something?

Reply:
Ah I see Dave already asked that elsewhere -- ignore this one.